src/box/lua/config/instance_config.lua

```diff
@@ -358,6 +360,17 @@ local function feedback_validate(data, w)
     w.error('Tarantool is built without feedback reports sending support')
 end
 
+local function audit_apply_default_if(_data, _w)
+    return has_audit
+end
+
+local function audit_validate(data, w)
+    if data == nil or has_audit then
+        return
+    end
+    w.error('Module audit is not available')
```

Comment:
The module is part of Tarantool EE. I would raise the same error as for other Enterprise Edition options. Moreover, with the `enterprise_edition = true` annotation the options will be listed in `--help-env-list` as available only in the Enterprise Edition. It should reduce a possible confusion.

Reply:
Thank you for the review! Done.